### PR TITLE
Gf fixes

### PR DESCRIPTION
**Pull request by ProjGoodF**
1. Edit package.json, script "start" did not work
2. GitHub Issue #266  _document.js file edited according to documentation next.js (https://github.com/zeit/next.js/blob/master/errors/no-document-title.md). At the moment title put only in pages/index.js. We need to use this revision and wait for the repeated site index linking from Google.
3. GitHub Issue #273  The height of the each section sets strongly in pixels. While opening the root section the height was figured according to a formula (quantity of child elements * 31 px), herewith the quantity of elements of the active child section wasn’t considered. Now it’s corrected.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "dev": "node server.js",
     "build": "next build",
-    "start": "NODE_ENV=production node server.js",
```
Comment (shcheklein) [inline package.json]: does Heroku set it by default? it affects certain parts of the code we haven't had any problems running it locally before - I think "dev" key is made for that?

**Comment by shcheklein** [comment]
@ProjGoodF couple minor comments, otherwise looks great! I think if you put Fixed #273 it will close the ticket automatically when PR is merged.

File: package.json (hunk 1 above)
Comment (ProjGoodF) [reply package.json]: When trying to start up the node wrote that "NODE_ENV" isn't an internal or external command, an executable program or a batch file.

**Comment by ProjGoodF** [comment]
> What is the thing about 4 empty files at the very end of you open the "Files changed" tab?

IDE PHP Storm recorded some changes in these files, so they were included in the commit, I did not create them. Perhaps this is due to the installation of dependencies from package.json

File: package.json (hunk 1 above)
Comment (ProjGoodF) [reply package.json]: > does Heroku set it by default? it affects certain parts of the code we haven't had any problems running it locally before - I think "dev" key is made for that?

When trying to start up the node wrote that "NODE_ENV" isn't an internal or external command, an executable program or a batch file.

Comment (shcheklein) [reply package.json]: My point is that this env variable is set there for a reason in the production environment. We need to make sure that we are not breaking the production environment behavior with this change (Heroku is where it's being deployed to). 

Are you running this on Windows? Would `npm run-script dev` be enough?

**Pull request closed without merging**